### PR TITLE
Inspec aws

### PR DESCRIPTION
**Pull request by fauzigo**
Description about what this pull request does.

Please make sure to follow the [DEV guidelines](https://gen3.org/resources/developer/dev-introduction/) before asking for review.

### New Features
- Burwood's implementation of AWS inspect.

### CODE REVIEW

File: inspec/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM     centos:centos7
```
Comment (frickjack) [inline inspec/docker/Dockerfile]: Is this the Dockerfile that we'll run as a cron job on the admin VM's?  Can we register it with quay, and make sure it builds?

File: inspec/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
```
Comment (frickjack) [inline inspec/docker/Dockerfile]: Maybe switch to an ubuntu 18 base image?  I feel like we know ubuntu pretty well ...

File: inspec/scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+The test are underneath the control directory with .rb extension.
+
+I have set this up two ways to run, the first way is to run a cronjob from an EC2 instance using the bash script and once the test is executed the results will be uploaded to an s3 bucket.
+The second way to execute the Chef Inspec test is to create a Kubernetes Cron Job to run on EKS cluster per environment. I created a Dockerfile using Centos7 some refactoring could be done to make the image smaller. Initially I set this up to run by injecting credentials using a Kubernetes secret object mounted as a read-only volume in order for the profile to be able to execute the test in a specific environment. You can create an IAM role as well in order to execute the Inspec test.
+With the Dockerfile you need to create a docker image for your ECR repository and then add that image into the Kubernetes Cron Job Yaml and deploy the yaml to create the Kubernetes object.
+
+
+VPC Flow Logs
+
```
Comment (frickjack) [inline inspec/scripts/README.md]: Can we put `#` and `##` markers before the section and subsection headings, so we get a nicely rendered page?

File: tf_files/gcp-bwg/roots/commons_setup/makefile [hunk 4]
```diff
@@ -16,8 +16,8 @@ all:
 	$(MAKE) -C org_setup plan && $(MAKE) -C org_setup apply; \
```
Comment (frickjack) [inline tf_files/gcp-bwg/roots/commons_setup/makefile]: Why are we changing the terraform files for GCP?  Has this been tested?

File: inspec/scripts/README.md [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+
+
+
+CloudTrail Logs
+
+Bash script to pull CloudTrail logs us just have to provide the values in the script Bucket Name Account Number and Region
+
+
+CloudTrail Lambda Function
```
Comment (frickjack) [inline inspec/scripts/README.md]: Is there automation for this, or are we not going to deploy this yet?  I would expect to see some terraform code ...

File: inspec/docker/Dockerfile (hunk 1 above)
Comment (fauzigo) [reply inspec/docker/Dockerfile]: Already taken care of https://quay.io/repository/cdis/inspec-aws

File: inspec/docker/Dockerfile (hunk 2 above)
Comment (fauzigo) [reply inspec/docker/Dockerfile]: maybe, but it'll be taken care of on a different branch.

Comment (fauzigo) [reply inspec/docker/Dockerfile]: maybe, but it'll be taken care of on a different branch.

File: inspec/scripts/README.md (hunk 5 above)
Comment (fauzigo) [reply inspec/scripts/README.md]: I'm unsure if this is something that'll be deployed any time soon, this is something the security team might be able to help out with.

File: inspec/scripts/README.md (hunk 3 above)
Comment (fauzigo) [reply inspec/scripts/README.md]: We can try

**Comment by fauzigo** [comment]
A new PR will be requested

**Pull request closed without merging**